sdk/big-mod-exp/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "solana-big-mod-exp"
+description = "Solana big integer modular exponentiation"
+documentation = "https://docs.rs/solana-big-mod-exp"
+version = { workspace = true }
+authors = { workspace = true }
+repository = { workspace = true }
+homepage = { workspace = true }
+license = { workspace = true }
+edition = { workspace = true }
+
+[target.'cfg(target_os = "solana")'.dependencies]
+solana-define-syscall = { workspace = true }
+
+[target.'cfg(not(target_os = "solana"))'.dependencies]
+num-bigint = { workspace = true }
+num-traits = { workspace = true }
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+getrandom = { workspace = true, features = ["js", "wasm-bindgen"] }
+js-sys = { workspace = true }
+wasm-bindgen = { workspace = true }
```

Comment:
Sorry if this is dense, but why are these here? I don't see anything related to wasm in this package